Documentation/distributions.md

```diff
@@ -145,17 +145,19 @@ Currently the rkt upstream project does not maintain its own repository, so user
 upgrade manually.
 
 ### rpm-based 
+**Note:** the example keyserver argument should be omitted if you already have a keyserver configured or you do not trust the SKS Keyservers.
 ```
-gpg --recv-key 18AD5014C99EF7E3BA5F6CE950BDD3E0FC8A365E
+gpg --keyserver hkp://p80.pool.sks-keyservers.net:80 --recv-key 18AD5014C99EF7E3BA5F6CE950BDD3E0FC8A365E
```

Comment:
Isn't `hkp://pool.sks-keyservers.net` enough? That seems to be recommended URL from https://sks-keyservers.net/ homepage.

Reply:
My experience is accessing port 80 causes less issues for drive-by users.  You can really pick the pool that scratches your itch:
https://sks-keyservers.net/overview-of-pools.php